code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2279,15 +2279,16 @@ All effects don't start immediately, but rather get worse over time; the rate is
 ////////////////////
 /datum/reagent/consumable/ethanol/species_drink
 	var/species_required
-	var/disgust = 25
+	var/disgust = 26
 	boozepwr = 50
 
-/datum/reagent/consumable/ethanol/species_drink/on_mob_life(mob/living/carbon/C)
-	if(C.dna.species && C.dna.species.species_category == species_required) //species have a species_category variable that refers to one of the drinks
-		quality = RACE_DRINK
-	else
-		C.adjust_disgust(disgust)
-	return ..()
+/datum/reagent/consumable/ethanol/species_drink/reaction_mob(mob/living/carbon/C, method=TOUCH)
+	if(method == INGEST)
+		if(C.dna.species && C.dna.species.species_category == species_required) //species have a species_category variable that refers to one of the drinks
```

Comment:
minor nitpick but if this isn't fixed here i'll do it myself later
```suggestion
		if(C?.dna?.species?.species_category == species_required) //species have a species_category variable that refers to one of the drinks
```

Reply:
Not a bad idea, I'll toss in your suggestion.